hypha/apply/projects/templates/application_projects/vendor_success.html

```diff
@@ -0,0 +1,23 @@
+{% extends "base-apply.html" %}
+{% load static %}
+
+{% block title %}Contractor Setup Completed{% endblock %}
+{% block content %}
+<div class="admin-bar">
+    <div class="admin-bar__inner">
+        <h2 class="heading heading--no-margin">Contractor Setup Completed</h2>
+    </div>
+</div>
+<div class="wrapper wrapper--small">
+    <h3>Thank you for submitting your information.</h3>
+    <p>Your OTF Programme Manager will be in touch with you if there are any issues.</p>
```

Comment:
Can we use {{ ORG_SHORT_NAME }} from  https://github.com/HyphaApp/hypha/blob/60c90d888f24af87f6644d62f145704cc08fe745/hypha/public/utils/context_processors.py#L15 instead of hard coding "OTF" here.